src/settings/settings.go

```diff
@@ -17,6 +17,7 @@ type Settings struct {
 	UseStatsd                    bool          `envconfig:"USE_STATSD" default:"true"`
 	StatsdHost                   string        `envconfig:"STATSD_HOST" default:"localhost"`
 	StatsdPort                   int           `envconfig:"STATSD_PORT" default:"8125"`
+	StatsdProtocol               string        `envconfig:"STATSD_PROTOCOL" default:"tcp"`
```

Comment:
Does setting STATSD_PROTOCOL without making this change not work for you?

(For me it seems to work without this change, just with the declaration in https://github.com/lyft/gostats/blob/f9add79ec5fce9579fb2e52b6b419da2ee965b37/settings.go#L26)

Reply:
Although even if it works without this, declaring it here might be useful from a documentation / discoverability perspective

Reply:
I did try setting `STATSD_PROTOCOL` and thought that it did not work which led to this PR. I will try it again, I probably made an error.

Reply:
Sorry, the issue seems to be that `STATS_PROTOCOL` does not work with the `envoyproxy/ratelimit:v1.4.0` container. Building a new container from main works without this change.

Would you like me to close it or leave it for documentation purposes?

Reply:
I don't have strong options either way, but also I'm not a maintainer so I wouldn't be able to merge regardless

Seems like removing the statsd stuff (host, port, etc.) from settings.go and just linking from the README to whatever documentation exists in the gostats library would require the least amount of ongoing maintenance

Reply:
+1 can we just move this to the README?